src/storage/index.ts

```diff
@@ -30,18 +30,21 @@ export class Storage {
     this.encryption = encryption ? new WebCrypto() : new DummyEncryption()
   }
 
-  async store(order: Order): Promise<string> {
-    return this.backend.store(await this.encryption.encrypt(this.encode(order)))
+  async store(order: Order, publicKey: JsonWebKey = null): Promise<string> {
+    return this.backend.store(
+      await this.encryption.encrypt(this.encode(order), publicKey),
+    )
   }
-  async load(id: string): Promise<Order> {
+  async load(id: string, privateKey: JsonWebKey = null): Promise<Order> {
     return this.decode(
-      await this.encryption.decrypt(await this.backend.load(id)),
+      await this.encryption.decrypt(await this.backend.load(id), privateKey),
     )
   }
-  async list(): Promise<Order[]> {
+  async list(privateKey: JsonWebKey = null): Promise<Order[]> {
```

Comment:
Same here: Why is `privateKey` set to `null` by `default`?

Reply:
See conversation above.